dashboard/src/components/Graph.js

```diff
@@ -1,14 +1,29 @@
 import React from "react";
+import { Nav, NavItem } from "react-bootstrap";
 
 import Label from "../components/Label";
 import ProgressBarContainer from "../containers/ProgressBarContainer";
 
 import "../assets/css/Graph.css";
 
 function Graph(props) {
-    let { text, success, plotAxis, fs, isFetching } = props;
-    let graphClass = fs ? "Graph-fs" : "Graph-s";
-    let bgColor;
+    let {
+        text,
+        success,
+        plotAxis,
+        fs,
+        isFetching,
+        json,
+        selectTab,
+        selectedTab
+    } = props,
+        graphClass = fs ? "Graph-fs" : "Graph-s",
+        bgColor,
+        hourglass = isFetching ? "Graph-hourglass" : "",
+        graphHeight = fs ? "Graph-full-height" : "Graph-fixed-height",
+        showGraph = selectedTab === "1" ? "" : "Graph-hide",
+        showJSON = selectedTab === "2" ? "" : "Graph-hide";
```

Comment:
Using https://github.com/JedWatson/classnames, we can simplify the optional class name.

    className={classnames({'Graph-hide': !showJSON});

If handling optional class names this way proves to be a pain, we can start using that package.